InternshipBE/BL/DTOs/AuthTokenDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BL.DTOs
+{
+	public class AuthTokenDTO
```

Comment:
Is it necessary?

Reply:
We have deleted this class. If FE team will say that they need some information with JWT token, then we can create this class with new properties.